doc/adding-custom-policies.md

```diff
@@ -94,6 +94,14 @@ oc create secret generic custom-policy-example-1 \
   --from-file=./example.lua
 ```
 
+By default, content changes in the secret will not be noticed by the 3scale operator.
+The 3scale operator allows monitoring the secret for changes adding the `apimanager.apps.3scale.net/watched-by=apimanager` label.
```

Comment:
```suggestion
The 3scale operator allows the monitoring of secret changes, this can be achieved by adding the `apimanager.apps.3scale.net/watched-by=apimanager` label to the required secret.
```

Reply:
updated. Thank you